packages/nvidia_nat_core/src/nat/front_ends/fastapi/http_interactive_runner.py

```
@@ -228,74 +228,68 @@ async def _streaming_generator_impl(
+        async def _acquire_session_and_push_workflow() -> None:
+            try:
+                async with self._session_manager.session(
+                        http_connection=request,
+                        user_input_callback=hitl_cb,
+                        user_authentication_callback=auth_cb,
+                ) as session:
+                    workflow_gen = workflow_gen_factory(session)
                     try:
                         async for item in workflow_gen:
                             await stream_queue.put(item)
                     except Exception as exc:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)